### DISCUSSION TITLE
Posibility for setting form conditions

### DISCUSSION THREAD

**Discussion by roelofsdevries** [nocodb/nocodb#4403] [Q&A] [answered]
Would anyone know if there exists a possibility in NocoDB for setting conditions for certain entries in form view? Would be very interesting if I could use conditional logic to show or hide certain fields based on the entry in previous fields. Thus far, I haven't managed to find a way to do so. 

Many thanks in advance!

**Comment by dstala** [comment, marked as answer]
Not possible currently

**Reply by roelofsdevries** [reply]
Thanks for your prompt reply! Is this feature scheduled in the upcoming future?

**Reply by tjoneslo** [reply]
I too would be interested in this feature.
